packages/gestalt/src/Box.js

```diff
@@ -279,7 +280,9 @@ const transformNumberOrPassthrough = (selector: string) => (
   m: Margin
 ): Style => {
   if (typeof m === 'number') {
-    return bind(rangeWithoutZero(selector), whitespace)(m);
+    return /^sd|md|lg/.test(selector)
```

Comment:
@AlbertCarreras does this need to be `sm` instead of `sd`? Would be great to add a few tests.

Also, do we need the regex check? Can we just do `return bind(rangeWithZero(selector), whitespace)(m);` which would also allow `margin={0}` to be supported?

Reply:
Following our previous conversation, passing all instances to **rangeWithZero** to support default margins with 0 point values.